### ISSUE TITLE
Status of the "manifest"

### ISSUE THREAD

**Issue by jogibear9988** [webcomponents/custom-elements-manifest#94]
@justinfagnani 

What is the status of the manifest?
Are we at a stable v1.0?
What is about the open issues? Many uncommented...
Also the open Pulls, whats the state?
What is about "webcompoents.org"? Wasn't there a plan to rebuild the site with elements using this spec?
What is about google's own elements (like material web components), will they create a manifest?

**Comment by sonntag-philipp** [comment]
I intended to use this manifest as it was already contained in the starter projects of lit. After taking a look at the open issues and evaluating the json schema directly, I don't think it will be reliable.

Instead I think that [JetBrains/Web-Types](https://github.com/JetBrains/web-types) will be a more reliable foundation for further development, even though I hope that both of those standards will be merged into a single set of definitions.

To be clear, my goal is not to downplay the effort put in this manifest but much more to emphasize that new adopters of this standard are turning away from it even before trying just due to the extremely unreliable maintenance.

**Comment by thepassle** [comment]
The schema is at a stable v1, we see a lot of orgs/companies (like Adobe, Shoelace, ING, and others) adopting the schema, and npm downloads steadily going up 🙂 

Consensus on the schema takes a lot of work and time and effort and discussion, so since the stable v1 this has naturally slowed down a bit.

**Comment by jogibear9988** [comment]
@thepassle yes, but I think in the manifest are a few big parts missing (the most problematic for me is: https://github.com/webcomponents/custom-elements-manifest/issues/96).
As I'm workin on a designer application (https://github.com/node-projects/web-component-designer), where I like to a components only by adding npm packages, I need information wich is currently missing in the manifest.
